src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -237,45 +243,103 @@ class TextToMiniTest {
         |module A
         |  axiom \tr ue() []
         |endmodule []
-      """.stripMargin
+        |""".stripMargin
     try {
       parseFromString(s)
       assert(false)
     } catch {
-      case _: Throwable => ()
-        val msg =
+      case ParseError(msg) =>
+        assertEquals(
           """
             |ERROR: Line 4: Column 12: Expected 'u', but ' '
             |  axiom \tr ue() []
             |           ^
-          """.stripMargin
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+          msg)
+    }
+  }
+
+  @Test def parseTestFail10(): Unit = {
+    val s =
+      """
+        |[]
+        |module A
+        |  axiom \t rue() []
+        |endmodule []
+        |""".stripMargin
+    try {
+      parseFromString(s)
+      assert(false)
+    } catch {
+      case ParseError(msg) =>
+        assertEquals(
+          """
+            |ERROR: Line 4: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but 't '
+            |  axiom \t rue() []
+            |          ^
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+          msg)
+    }
+  }
+
+  @Test def parseTestFail11(): Unit = {
+    val s =
+      """
+        |[]
+        |module A
+        |  axiom \t
+        |""".stripMargin
+    try {
+      parseFromString(s)
+      assert(false)
+    } catch {
+      case ParseError(msg) =>
+        assertEquals(
+          """
+            |ERROR: Line 4: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but 't\n'
+            |  axiom \t
+            |          ^
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+          msg)
```

Comment:
I added more tests to increase the test coverage, but still not 100%. Having 100% is impossible or nonsense, partly due to scala-specific issues of intellij coverage measurement, and conservative error handling codes that are not possible to reach for now. If you can find uncovered codes that should be covered, let me know.